js/app/src/Index.svelte

```diff
@@ -158,12 +158,12 @@
 
 			if (parsed_head_html) {
 				for (let head_element of parsed_head_html) {
-					let newScriptTag = document.createElement("script");
+					let newElement = document.createElement(head_element.tagName);
```

Comment:
Also included a quick fix here. I was incorrectly assuming, the user would always pass a `<script>` tag.